android/build.gradle

```diff
@@ -49,6 +49,7 @@ allprojects {
 
         google()
         jcenter()
+        maven { url 'https://jitpack.io' }
```

Comment:
I think we can skip this. The PR doesn't give any further information, and searching the web I don't see a lot of evidence of people using this service.

It'll be easy to add a line like this in the future as part of adopting any library that is published there.